tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers.ts

```diff
@@ -469,6 +478,37 @@ const reducer = createReducer(
       return newState;
     }
   ),
+  on(
+    actions.sourceFileListRequested,
+    (state: DebuggerState): DebuggerState => {
+      return {
+        ...state,
+        sourceCode: {
+          ...state.sourceCode,
+          sourceFileListLoaded: {
+            ...state.sourceCode.sourceFileListLoaded,
+            state: DataLoadState.LOADING,
+          },
+        },
+      };
+    }
+  ),
+  on(
+    actions.sourceFileListLoaded,
+    (state: DebuggerState, sourceFileList): DebuggerState => {
+      return {
+        ...state,
+        sourceCode: {
+          ...state.sourceCode,
+          sourceFileListLoaded: {
+            state: DataLoadState.LOADED,
```

Comment:
nit: since L489 adds 
```
sourceFileListLoaded: {
            ...state.sourceCode.sourceFileListLoaded,
```
we should probably add the same thing here for safety?

Reply:
Done.